src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/auth/AuthApiService.kt

```diff
@@ -70,6 +70,12 @@ class AuthApiService(
       ),
     )
 
+  fun recognised(username: String) =
```

Comment:
better name might be usreRecognised - but not a biggie

Reply:
Happy to change, but kind of like it being succinct where the usage context makes it obvious - felt like it would in this case as we're only passing in the user name. Happy to change though

Reply:
This is now called confirmRecognised - please let me know what you think